docs/source/graph-construction/index.rst

```diff
@@ -0,0 +1,18 @@
+==============================
+Distributed Graph Construction
+==============================
+
+Graphstorm offers options to do distributed graph construction in clusters.
+
+The following sections provide comprehensive guidance on effectively utilizing GSProcessing.
+The first section details the installation process for GSProcessing.
+The second section offers examples on drafting a configuration file for a GSProcessing job.
+The final section explains how to deploy your GSProcessing job into AWS infrastructure.
+
+.. toctree::
+   :maxdepth: 2
+   :glob:
+
+   gs-processing/gs-processing-prerequisites-index.rst
```

Comment:
Shall we move prerequisites to `gs-processing/developer/` ?
@thvasilo @jalencato

Reply:
I do not recommend to do that, if checking the content inside the ``gs-processing-prerequisites-index.rst``:

```
gs-processing-getting-started.rst
usage/distributed-processing-setup.rst
usage/example.rst
```
We need to refer the gs-processing-getting-started.rst, so I would recommend to put the index files here in the same level. And btw, it does not affect how it presents in the read-the-doc.

Reply:
I create another folder ``index`` to store all the gsprocessing related index here just for these rst files, I would say this will be a better way than storing the index files in the index or usage folder.